tests/performance/producer/stress.js

```diff
@@ -0,0 +1,80 @@
+export * from "./client.js";
+
+/*
+ * NOTE - To run the producer K6 tests, you need to prepare the data by setting
+ * the output_full_pointers flag to true in the environment.py:setup() method.
+ */
+
+export const options = {
+  tlsAuth: [
+    {
+      cert: open(`../../../truststore/client/${__ENV.ENV_TYPE}.crt`),
+      key: open(`../../../truststore/client/${__ENV.ENV_TYPE}.key`),
+    },
+  ],
+  scenarios: {
+    createDocumentReference: {
+      exec: "createDocumentReference",
+      executor: "ramping-vus",
+      startVUs: 1,
+      stages: [
+        { target: 10, duration: "30s" },
+        { target: 10, duration: "1m" },
+      ],
+    },
+    readDocumentReference: {
+      exec: "readDocumentReference",
+      executor: "ramping-vus",
+      startVUs: 1,
+      stages: [
+        { target: 10, duration: "30s" },
+        { target: 10, duration: "1m" },
+      ],
+    },
+    updateDocumentReference: {
+      exec: "updateDocumentReference",
+      executor: "ramping-vus",
+      startVUs: 1,
+      stages: [
+        { target: 10, duration: "30s" },
+        { target: 10, duration: "1m" },
+      ],
+    },
+    deleteDocumentReference: {
+      exec: "deleteDocumentReference",
+      executor: "ramping-vus",
+      startVUs: 1,
+      stages: [
+        { target: 10, duration: "30s" },
+        { target: 10, duration: "1m" },
+      ],
+    },
+    upsertDocumentReference: {
+      exec: "upsertDocumentReference",
+      executor: "ramping-vus",
+      startVUs: 1,
+      stages: [
+        { target: 10, duration: "30s" },
+        { target: 10, duration: "1m" },
+      ],
+    },
+    /*searchDocumentReference: {
```

Comment:
I'd rather these just be deleted if they're not going to be uncommented in the near future

Reply:
yeah, good shout, I was on the fence about this but happy to remove as there's no guarantee these will be re-enabled in the short term (we need to fix the endpoints and the tests first)